security/pkg/stsservice/test/sts_fetch_timeout/sts_timeout_test.go

```diff
@@ -65,7 +65,7 @@ func TestTokenFetchTimeoutTwo(t *testing.T) {
 	t.Skip("https://github.com/istio/istio/issues/20133")
 	cb := xdsService.CreateXdsCallback(t)
 	// Start all test servers
-	setup := stsTest.SetUpTest(t, cb, testID.STSTimeoutTest)
+	setup := stsTest.SetUpTest(t, cb, testID.STSTimeoutTest, false)
```

Comment:
nit: here and everywhere: the casing of setup in functions names like SetUp and setUp is annoying :P , even if not addressed by this PR, could have been fixed.